src/httpfetch.h

```diff
@@ -35,6 +35,13 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 // Start of regular allocated caller IDs.
 #define HTTPFETCH_CID_START 3
 
+namespace {
+	// lower bound for curl_timeout (see also settingtypes.txt)
+	constexpr long MIN_HTTPFETCH_TIMEOUT_INTERACTIVE = 1000;
```

Comment:
it's a static const, not a constexpr, there is no calculation

Reply:
`static const` forces the compiler to write it into a memory section, this is avoided with `constexpr`.